### PR TITLE
StorageHub type support

### PR DESCRIPTION
**Pull request by timbrinded**
added StorageHub local endpoints and types

### CODE REVIEW

File: packages/apps-config/src/api/spec/storagehub.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2017-2024 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { storageHubDefinitions } from '@storagehub/types-bundle';
+
+export default storageHubDefinitions;
```
Comment (ffarall) [inline packages/apps-config/src/api/spec/storagehub.ts:6]: Why is this needed and where is `storageHubDefinitions` coming from?

**Review by ffarall — APPROVED** [review]

File: packages/apps-config/src/api/spec/storagehub.ts (hunk 1 above)
Comment (timbrinded) [reply packages/apps-config/src/api/spec/storagehub.ts:6]: This is importing all the manual definitions from the storageHub repo into polkadotJS. this way we wont have to keep changing this repo, and we can just up the package ver when our rpcs change
